<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createmeal_basicparams</class>
 <widget class="QDialog" name="createmeal_basicparams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Meal</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
   <property name="placeholderText">
    <string>enter a name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_duration">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Days between occurrences</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_daysbetweenoccurrences">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;A name for your meal, such as: Pizza, Tacos, Ceasar Salad, etc.&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Price&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;The cost for this meal.&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Duration&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;How many days does this meal last? &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;(for meal prepping, otherwise 1 is typical)&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Days between occurrences &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Limit how many times this meal may occur. &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;(0 means no delay. 2 means there must be a gap of at least 2 days before the meal may occur again)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
